crates/dht/src/time.rs

```diff
@@ -742,26 +750,34 @@ mod tests {
         let store = Arc::new(Kitsune2MemoryOpStore::default());
         store
             .process_incoming_ops(vec![
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![7; 32])),
-                    timestamp: origin_timestamp,
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![23; 32])),
-                    timestamp: origin_timestamp,
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![7; 32])),
-                    timestamp: (now - UNIT_TIME).unwrap(),
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![29; 32])),
-                    timestamp: (now - UNIT_TIME).unwrap(),
-                    op_data: vec![],
-                },
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![7; 32])),
+                    origin_timestamp,
+                    vec![],
+                )
+                .try_into()
+                .unwrap(),
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![23; 32])),
+                    origin_timestamp,
+                    vec![],
+                )
+                .try_into()
+                .unwrap(),
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![11; 32])),
```

Comment:
Note the change from 7 to 11, reusing the same op id isn't a good idea and this test broke with the changed op store

Reply:
In which way did it break?

Reply:
The hash comparison later in the test (checking that hashes got XORed together) failed. Because the later 7 overwrote the original